micro-rdk/src/esp32/mod.rs

```diff
@@ -6,16 +6,20 @@ pub mod board;
 pub mod camera;
 pub mod certificate;
 pub mod dtls;
+#[cfg(feature = "builtin-components")]
```

Comment:
can you group them in brackets?

Reply:
I actually tried, it got mad at me

Reply:
to clarify, this yields a syntax error
```rust
#[cfg(feature = "builtin-components")]
{
    pub mod encoder;
    pub mod hcsr04;
    pub mod single_encoded_motor;
    pub mod single_encoder;
}
```
and I'm struggling to find another way to do this

Reply:
I went looking for a way to do this too and failed. So much for `M-x sort-lines` on this block!